docs/README.md

```diff
@@ -1,121 +1,240 @@
+</div>
+
+<div id="TPC-C_3" class="tabcontent" markdown="1">
+
+| Name                   | Description                                                                                                                                                           | Default   |
+|:-----------------------|:----------------------------------------------------------------------------------------------------------------------------------------------------------------------|:----------|
+| `num_warehouses`       | Number of warehouses (scale factor) for benchmarking.                                                                                                                 | `1`       |
+| `load_concurrency`     | Number of threads for loading.                                                                                                                                        | `1`       |
+| `load_start_warehouse` | Start ID of loading warehouse. This option can be useful with `--skip-item-load` when loading large-scale data with multiple clients or adding additional warehouses. | `1`       |
+| `load_end_warehouse`   | End ID of loading warehouse. You can use either `--num-warehouses` or `--end-warehouse` to specify the number of loading warehouses.                                  | `1`       |
+| `skip_item_load`       | Whether or not to skip loading item table.                                                                                                                            | `false`   |
+| `use_table_index`      | Whether or not to use a generic table-based secondary index instead of ScalarDB's secondary index.                                                                    | `false`   |
+| `np_only`              | Run benchmark with only new-order and payment transactions (50% each).                                                                                                | `false`   |
+| `rate_new_order`       | Percentage of new-order transactions. Specify all the rate parameters to use your own mix.                                                                            | N/A       |
+| `rate_payment`         | Percentage of payment transactions. Specify all the rate parameters to use your own mix.                                                                              | N/A       |
+| `rate_order_status`    | Percentage of order-status transactions. Specify all the rate parameters to use your own mix.                                                                         | N/A       |
+| `rate_delivery`        | Percentage of delivery transactions. Specify all the rate parameters to use your own mix.                                                                             | N/A       |
+| `rate_stock_level`     | Percentage of stock-level transactions. Specify all the rate parameters to use your own mix.                                                                          | N/A       |
```

Comment:
Does `Specify all the rate parameters to use your own mix` mean the following?:

```suggestion
| `rate_new_order`       | Percentage of new-order transactions. Specify this percentage based on your needs. The total of all rate parameters must be 100 percent.                                                                            | N/A       |
| `rate_payment`         | Percentage of payment transactions. Specify this percentage based on your needs. The total of all rate parameters must be 100 percent.                                                                              | N/A       |
| `rate_order_status`    | Percentage of order-status transactions. Specify this percentage based on your needs. The total of all rate parameters must be 100 percent.                                                                         | N/A       |
| `rate_delivery`        | Percentage of delivery transactions. Specify this percentage based on your needs. The total of all rate parameters must be 100 percent.                                                                             | N/A       |
| `rate_stock_level`     | Percentage of stock-level transactions. Specify this percentage based on your needs. The total of all rate parameters must be 100 percent.                                                                          | N/A       |
```

Or if the total can be less than 100 percent, maybe the following is better?:

```suggestion
| `rate_new_order`       | Percentage of new-order transactions. Specify this percentage based on your needs. The total of all rate parameters must be 100 percent or less.                                                                            | N/A       |
| `rate_payment`         | Percentage of payment transactions. Specify this percentage based on your needs. The total of all rate parameters must be 100 percent or less.                                                                              | N/A       |
| `rate_order_status`    | Percentage of order-status transactions. Specify this percentage based on your needs. The total of all rate parameters must be 100 percent or less.                                                                         | N/A       |
| `rate_delivery`        | Percentage of delivery transactions. Specify this percentage based on your needs. The total of all rate parameters must be 100 percent or less.                                                                             | N/A       |
| `rate_stock_level`     | Percentage of stock-level transactions. Specify this percentage based on your needs. The total of all rate parameters must be 100 percent or less.                                                                          | N/A       |
```

Reply:
The former one is correct, but 'all' means that users must specify all rate_x parameters when one of them is specified. So, I think only the sentence `The total of all rate parameters must be 100 percent.` might lose that kind of perspective.

Reply:
Great, perfect!